ARG queries/Container Registry Vulnerability Assessment Images Scan Summary/readme.md

```diff
@@ -0,0 +1,20 @@
+# Container Registry Images scan vulnerability assessment summary
+**Easy way to get Images summary for vulnerabilities scan, can be filtered for specific image or repository.**
+
+Azure Resource Graph (ARG) provides an efficient way to query at scale across a given set of subscriptions for any Azure Resource (for more information please visit https://docs.microsoft.com/en-us/azure/governance/resource-graph/).
+```
+
+securityresources
+ | where type == 'microsoft.security/assessments/subassessments'
+ | where id matches regex  '(.+?)/providers/Microsoft.Security/assessments/dbd0cb49-b563-45e7-9724-889e799fa648/'
```

Comment:
Suggest to use "parse" operator instead. for example:
`
| parse id with RegistryResourceId "/providers/Microsoft.Security/assessments/" *
| parse RegistryResourceId with * "/providers/Microsoft.ContainerRegistry/registries/" RegistryResourceName
`